### ISSUE TITLE
No clear way to delete a document

### ISSUE THREAD

**Issue by ogoldberg** [neo4j-labs/arrows.app#56]
I have a bunch of blank documents that I'd like to delete, but there doesn't seem to be any way to do that.
Because of this I went to my google drive to find the documents and delete them there, but I couldn't find them.

**Comment by apcj** [comment]
Hi @ogoldberg it's funny that you can't locate the documents in your Google Drive. I can see the documents I create in the root folder at https://drive.google.com/drive/my-drive . Does the header of arrows.app say "Saved to Google Drive"?

**Comment by bamalburg** [comment]
I also agree with this issue, at least with web browser storage - I've found a way to delete them, but it requires doing this:
Developer Tools > Application tab > Local Storage section > click on https://arrows.app > clicking on the Value column > looking at the "diagramName" field for one that I want to delete and deleting it (and also clicking on the row where the value in the "Key" column is "neo4j-arrows-app.recentlyAccessedDiagrams" and deleting that row so the deleted diagram's name doesn't show up in my recently accessed diagrams, if it was previously there)

[Caveats: what I did may be wrong, and/or a bad idea, and/or I am missing some easier way to do it, etc.]

**Comment by akollegger** [comment]
Storage behaviors and interactions are worth reconsidering, particularly as we think about establishing a well-defined extension point to make storage pluggable both for save/load and import/export. 

-ABK
